server/src/main/java/org/candlepin/guice/CandlepinModule.java

```diff
@@ -448,6 +449,7 @@ private void configureAsyncJobs() {
         JobManager.registerJob(UnmappedGuestEntitlementCleanerJob.JOB_KEY,
             UnmappedGuestEntitlementCleanerJob.class);
         JobManager.registerJob(ActiveEntitlementJob.JOB_KEY, ActiveEntitlementJob.class);
+        JobManager.registerJob(AsyncJobCleaner.JOB_KEY, AsyncJobCleaner.class);
```

Comment:
We try to keep the list of registrations alphabetical here for readability, so it would be consistent to move this and the `ActiveEntitlementJob` above it to the start of the order.

Reply:
Done.